.github/workflows/firebase-hosting-merge.yml

```diff
@@ -0,0 +1,25 @@
+# This file was auto-generated by the Firebase CLI
+# https://github.com/firebase/firebase-tools
+
+name: Deploy to Firebase Hosting on merge
+"on":
+  push:
+    branches:
+      - main
+      
+env:
+  REACT_APP_apiKey: ${{secrets.AIzaSyB4I_Ks4iCbnywjv-3VcPQ-4mCrW1Pd3I8}}
+
+jobs:
+  build_and_deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: FirebaseExtended/action-hosting-deploy@v0
+        with:
+          repoToken: "${{ secrets.GITHUB_TOKEN }}"
+          firebaseServiceAccount: "${{ secrets.FIREBASE_SERVICE_ACCOUNT_RUSTEE_PROD }}"
```

Comment:
Can we rename this to `FIREBASE_SERVICE_ACCOUNT` instead? I think RUSTEE_PROD is being attached due to your project id.